File: Ruby/Fizzbuzz.rb
for num in (1..100) do
    if num.modulo(15)==0
        puts "FizzBuzz"
    elsif num.modulo(5)==0
        puts "Buzz"
    elsif num.modulo(3)==0
        puts "Fizz"
    else 
        puts num
    end
end

File: Elixir/fizz_buzz.exs
Enum.map(1..100, fn n ->
  cond do
    rem(n, 3) == 0 and rem(n, 5) == 0 -> "FizzBuzz"
    rem(n, 3) == 0 -> "Fizz"
    rem(n, 5) == 0 -> "Buzz"
    true -> n
  end
end)
|> Enum.each(&IO.puts(&1))
